src/com/epam/collections/models/Student.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.collections.models;
+
+public class Student implements Comparable {
+    private int id;
+    private String firstName;
+    private String lastName;
+    private int age;
+
+    public Student(int id, String firstName, String lastName, int age) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.age = age;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    @Override
+    public String toString() {
+        return this.firstName + " " + this.lastName +
+                " " + this.age;
+    }
+
+    @Override
+    public int compareTo(Object object) {
+        return this.lastName.compareTo(((Student) object).lastName);
+    }
+
+    @Override
+    public boolean equals(Object object) {
+        return this.getId() == ((Student) object).getId();
```

Comment:
Suppose I know there is a student whose name is Abraham Poghosyan and age is 20. I won't be able to remove him unless I know his id as well.

Reply:
Suppose  there are two students whose firstnames ,lastnames and ages are the same .Thats why i added ID ,to differ them.